CfStreamUploader/CfStreamUploader.Presentation/ViewModel.cs

```diff
@@ -1,8 +1,135 @@
 ﻿using GalaSoft.MvvmLight;
+using GalaSoft.MvvmLight.Command;
 
 namespace CfStreamUploader.Presentation
 {
     public class ViewModel : ViewModelBase
     {
+        #region ColorChange
+
+        private bool isDarkmode = false;
+        public string themeText = "Lightmode";
+
+        private string baseColor = "Transparent";
+        private string contrastColor = "Transparent";
+        private string textColor = "Black";
+        private string borderBrush = "#6497e8";
+        private string button1Bg = "#6497e8";
+        private string button1Fg = "White";
+        private string button2Bg = "#6497e8";
+        private string button2Fg = "#6497e8";
+        private string button2FgMouseOver = "White";
+        private string progressColor = "Green";
+
+        public string ThemeText
+        {
+            get => this.themeText;
+            set => this.Set(ref this.themeText, value);
+        }
+
+        public string BaseColor
+        {
+            get => this.baseColor;
+            set => this.Set(ref this.baseColor, value);
+        }
+
+        public string ContrastColor
+        {
+            get => this.contrastColor;
+            set => this.Set(ref this.contrastColor, value);
+        }
+
+        public string TextColor
+        {
+            get => this.textColor;
+            set => this.Set(ref this.textColor, value);
+        }
+
+        public string BorderBrush
+        {
+            get => this.borderBrush;
+            set => this.Set(ref this.borderBrush, value);
+        }
+
+        public string Button1Bg
+        {
+            get => this.button1Bg;
+            set => this.Set(ref this.button1Bg, value);
+        }
+
+        public string Button1Fg
+        {
+            get => this.button1Fg;
+            set => this.Set(ref this.button1Fg, value);
+        }
+
+        public string Button2Bg
+        {
+            get => this.button2Bg;
+            set => this.Set(ref this.button2Bg, value);
+        }
+
+        public string Button2Fg
+        {
+            get => this.button2Fg;
+            set => this.Set(ref this.button2Fg, value);
+        }
+
+        public string Button2FgMouseOver
+        {
+            get => this.button2FgMouseOver;
+            set => this.Set(ref this.button2FgMouseOver, value);
+        }
+
+        public string ProgressColor
+        {
+            get => this.progressColor;
+            set => this.Set(ref this.progressColor, value);
+        }
+
+        public RelayCommand SetDarkmodeCommand { get; }
+
+        private void SetDarkmode()
```

Comment:
outsource?

Reply:
SetDarkmode() uses the props in ViewModel.
this.SetDarkmodeCommand = new RelayCommand(this.SetDarkmode);
i don't think that we sould outsource the Method.